trulens_eval/trulens_eval/app.py

```diff
@@ -276,6 +276,7 @@ def _post_record(
         ret_record_args['cost'] = cost
```

Comment:
@joshreini1 are tags ever in documentation or tested? Can we add that?

Reply:
Added tag to quickstart example